soccer/gameplay/plays/sequences/move_sequence.py

```diff
@@ -0,0 +1,22 @@
+import behavior_sequence
+import skills.move
+
+class MoveSequence(behavior_sequence.BehaviorSequence):
+    def __init__(self, positions = None):
+        super().__init__()
+        self._positions = [] if positions == None else positions
+
+    def on_enter_start(self):
+        super().on_enter_start()
+        for pos in self.positions:
+            self.behaviors.append(skills.move.Move(pos))
+
+    @property
```

Comment:
this is what I am looking at, isn't working. It says I have no attribute "_positions" at line 16